### ISSUE TITLE
What's my version?

### ISSUE THREAD

**Issue by softwaregravy** [bploetz/versionist#49]
I'm not sure if this is a feature request or a help request.

I've decided to use versionist for my api routing and active_model_serializers to manage my presentation of my objects in the json object. I'm using https://github.com/hookercookerman/active_model_version_serializers so that I can version my model serialization (no context with that gem needed to understand this issue).

I have a common base class for all my controllers of a given version, and right now it looks like this:

```ruby
class Api::V20130701::BaseController < Api::ApiController
  def default_serializer_options
    {version: :V20130701}
  end 
end
```

The `version` arg there tells `active_model_version_serializers` what version to use unless told otherwise. It matches my version from my routes file.

I would love to be able to get the current versionist module from the request somehow. I would like to get this from a standard place. That is agnostic to what options I've set up versionist with (header vs url param). 

As an example, I'd be able to do something like:

```ruby
class Api::V20130701::BaseController < Api::ApiController
  def default_serializer_options
    {version: api_version}
  end 
end
```

Where `api_version` is a helper method added to `ActionController::Base`. I'm pretty indifferent to the mechanism used (params? env?), but I would like to get the version of the current request.

Thanks,

Adding routes.rb snippet just in case:

```ruby
  namespace :api do  
    api_version(:module => "V20130701", 
                :header => {:name => "API-VERSION", :value => "20130701"}, 
                :defaults => {:format => "json"}, 
                :default => true) do
```

**Comment by dan-corneanu** [comment]
Was there any resolution on this issue?

**Comment by bploetz** [comment]
@dan-corneanu Not really. I think it would be trivial to add something like this, but I'm still curious as to the use cases where you'd need this.

**Comment by softwaregravy** [comment]
The goal was just not to have the String in the routes file, and also in the class. It's a minor point, but requires having the same string in a couple of places. Basically, I've routed the request via a version number to a given controller. 

It's been a long time since I opened this. Since I never followed up answering Brian's very prompt response to my original question, it must not have been that important.
